internal/api/impl/v1/dashboard/service.go

```diff
@@ -42,47 +45,55 @@ func NewService(dao dashboard.DAO, globalVarDAO globalvariable.DAO, projectVarDA
 }
 
 func (s *service) Create(_ apiInterface.PersesContext, entity *v1.Dashboard) (*v1.Dashboard, error) {
+	copyEntity, err := deep.Copy(entity)
```

Comment:
To be safer and ensure we never use the original entity, you could encapsulate all the create logic into a create function and the copy would stay in the Create WDYT ?

Reply:
ah yes good idea